packages/server/src/automations/utils.js

```diff
@@ -0,0 +1,205 @@
+const env = require("../environment")
+const workerFarm = require("worker-farm")
+const { getAPIKey, update, Properties } = require("../utilities/usageQuota")
+const singleThread = require("./thread")
+const { definitions } = require("./triggerInfo")
+const webhooks = require("../api/controllers/webhook")
+const CouchDB = require("../db")
+const { queue } = require("./bullboard")
+const newid = require("../db/newid")
+const { updateEntityMetadata } = require("../utilities")
+const { MetadataTypes } = require("../constants")
+
+const WH_STEP_ID = definitions.WEBHOOK.stepId
+const CRON_STEP_ID = definitions.CRON.stepId
+
+let workers = workerFarm(require.resolve("./thread"))
+
+function runWorker(job) {
```

Comment:
Is `worker-farm` necessary anymore because we can assume redis locally and just use bull? IIRC `worker-farm` was a threading workaround for the electron app locally